package.json

```diff
@@ -36,14 +39,15 @@
     "qejs": "^3.0.5",
     "ractive": "^0.6.1",
     "react": "^0.14.6",
+    "react-dom": "^0.14.7",
     "react-tools": "^0.13.0",
```

Comment:
Yes, you are right. This two lines confused me: https://github.com/facebook/react/blob/master/package.json#L60-L63